src/simulate.jl

```diff
@@ -160,6 +164,7 @@ function simulate(JADEmodel::JADEModel, parameters::JADESimulation)
                     terminate_on_cycle = false,
                     terminate_on_dummy_leaf = false,
                 ),
+                parallel_scheme = async ? SDDP.Threaded() : SDDP.Serial(),
```

Comment:
```suggestion
                parallel_scheme,
```

Etc? Or maybe this is less clear and what you have is good...